EKF/control.cpp

```diff
@@ -326,7 +326,7 @@ void Ekf::controlExternalVisionFusion()
 		}
 
 	} else if (_control_status.flags.ev_pos
-		   && (_time_last_imu - _time_last_ext_vision > (uint64_t)_params.no_gps_timeout_max)) {
+		   && ((uint64_t)(std::llabs(_time_last_imu - _time_last_ext_vision)) > (uint64_t)_params.no_gps_timeout_max)) {
```

Comment:
I'm not sure if this does what you're intending. These are unsigned types. Do you also want to treat a sufficiently large future timestamp as a timeout as well?

What about a simple check that `_time_last_imu >= _time_last_ext_vision`?